content/docs/09.administrator-guide/02.deployment/04.restack.md

```diff
@@ -1,56 +1,190 @@
 ---
-title: Deployment on AWS with Restack
+title: Deployment on your AWS, GCP or Azure with Restack
 ---
 
-[Restack](https://restack.io) lets you deploy Kestra on your own AWS infrastructure on Kubernetes.
+[Restack](https://restack.io) provides the tools and infrastructure to build and deploy open-source products in your cloud without DevOps.
```

Comment:
```suggestion
[Restack](https://restack.io) provides the tools to deploy open-source products to your cloud without requiring DevOps resources.
```

It is a little confusing to say: it provides infrastructure and then "deploy to your Cloud"